task_sdk/src/airflow/sdk/__init__.py

```diff
@@ -31,11 +31,15 @@
     "dag",
     "get_current_context",
     "get_parsing_context",
+    "__version__",
+    "Asset",
+    "AssetWatcher",
```

Comment:
Nit: `__version__` is duplicated, and can you sort this list.

And we should probably enable https://docs.astral.sh/ruff/rules/unsorted-dunder-all/#unsorted-dunder-all-ruf022 to have ruff enforce and sort it for us.